src/tactics/FStar.Tactics.Basic.fst

```diff
@@ -585,6 +584,7 @@ let __tc_ghost (e : env) (t : term) : tac (term * typ * guard_t) =
     bind get (fun ps ->
     mlog (fun () -> BU.print1 "Tac> __tc_ghost(%s)\n" (Print.term_to_string t)) (fun () ->
     let e = {e with uvar_subtyping=false} in
+    let e = {e with letrecs=[]} in
```

Comment:
In some examples, the tactic was invoked in the context of a recursive function and had `env.letrecs` set, and then tactic called back into the typechecker with the same env, resulting in `letrecs is set but missing type annotations error`.